src/percy-agent-client/serialize-cssom.ts

```diff
@@ -0,0 +1,30 @@
+// Take all the CSS created in the CSS Object Model (CSSOM), and inject it
+// into the DOM so Percy can render it safely in our browsers.
+// Design doc:
+// https://docs.google.com/document/d/1Rmm8osD-HwSHRpb8pQ_1wLU09XeaCV5AqMtQihk_BmM/edit
+export function serializeCssOm(document: HTMLDocument) {
```

Comment:
you could have this function return a `Document`, like this

```typescript
export function serializeCssOm(document: HTMLDocument): Document {
```

And then above in `percy-agent.ts`, change the call to:

```typescript
const docWithSerializedCSSOM = serializeCssOm(doc)
```

Reply:
I just removed that type annotation in `percy-agent.ts`. (Had it there initially to make it clear that this is not just an `HTMLElement` the way `serializeInputElements` returns, but you're right that's it's probably unnecessary.)